packages/pages-components/src/components/analytics/interfaces.ts

```diff
@@ -1,7 +1,15 @@
-import { MouseEvent } from "react";
-import { ConversionDetails, Visitor } from "@yext/analytics";
+import { Action } from "@yext/analytics";
 import { TemplateProps } from "./types.js";
 
+export type TrackProps = {
+  action: Action;
+  scope?: string;
+  eventName?: string;
+  amount?: number;
+  currency?: string;
+  destinationUrl: string;
```

Comment:
this should be above the optional params

Reply:
Order doesn't matter in a type because you have to specify the keys when setting.